byway/src/main/java/com/google/sps/servlets/RecsServlet.java

```diff
@@ -38,34 +39,22 @@
   private final Gson gson = new Gson(); 
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
   
-  /** Fills datastore with hardcoded values for Recommendation objects,
-  values will be from another datastore later **/
-  @Override
+  /* Intializes a mock Trip entity
   public void init(){
-    // when using actual values, there will only be one Entity object instantiated, not one per stop
-    // loop through Rena and Leo's datastore entries for recommended stops
-    Entity recEntity1 = new Entity(Recommendation.KIND);
-    Entity recEntity2 = new Entity(Recommendation.KIND);
-    Entity recEntity3 = new Entity(Recommendation.KIND);
-    recEntity1.setProperty("placename", "Times Square");
-    datastore.put(recEntity1);
-    recEntity2.setProperty("placename", "MOMA");
-    datastore.put(recEntity2);
-    recEntity3.setProperty("placename", "Central Park");
-    datastore.put(recEntity3);
+    Entity tripEntity = new Entity(Trip.KIND);
+    tripEntity.setProperty("id", 1234);
+    datastore.put(tripEntity);
   }
+  **/
 
+  /* Passes hard-coded data to be shown in the recommendations panel
+   * TODO: Replace with actual arraylist of recommended places based on TextSearch in
+   * interests page 
+   */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query(Recommendation.KIND); 
-    PreparedQuery results = datastore.prepare(query);
-    List<Recommendation> recs = new ArrayList<>();
-
-    for (Entity entity: results.asIterable()){
-      recs.add(Recommendation.fromEntity(entity));
-    }
+    List<String> recs = new ArrayList<String>(Arrays.asList("Times Square", "MOMA", "Central Park" ));
```

Comment:
If you don't need to be able to modify the list, you can just directly use `Arrays.asList` as a `List`, rather than creating an `asList` and then making a copy of it to create an `ArrayList`.

Reply:
Yup, these are just hardcoded values. Changed.

Reply:
Huh, I changed this in commit df9a43e3dd6b7596c5f2fe925f89951d9bb8b70f.